.gitmodules

```diff
@@ -1,4 +0,0 @@
-[submodule "CMakeModules"]
-	path = CMakeModules
-	url = https://github.com/NOAA-EMC/CMakeModules
-	branch = develop
```

Comment:
@kgerheiser I added this .gitmodules back because otherwise, "build_stack.sh" from the hpc-stack umbrella will stop with errors at "git submodule update --init --recursive" (although standalone build has no problems).  I did not dig further on this issue. Maybe I have something wrongly configured. Hope it is fine with the latest hpc-stack without this .gitmodules.

Reply:
That is because the the CMakeModules folder was still there with no corresponding `.gitmodules`. I have deleted `CMakeModules` so everything is good now.

Reply:
That makes sense. Thanks!